.eslintrc.json

```diff
@@ -0,0 +1,14 @@
+{
+  "extends": [
+    "@lifeomic/standards/javascript"
```

Comment:
non blocking: add `prettier` extends and plugin, then add `prettier/prettier: error`